### PR TITLE
Update Pluggables Framework Documentation

### LINKED ISSUE #4735: Docs: Pluggables are not SSR friendly
> "Problem" with Pluggables is that they are not SSR friendly. So, they are good for CMSUI, or all client components/widgets, but not for data.

Got it. It should say that in the docs. ;-)

File to edit: [Pluggables framework](https://6.docs.plone.org/volto/recipes/pluggables.html?highlight=pluggable#)

_Originally posted by @davisagli in https://github.com/plone/volto/issues/4733#issuecomment-1533199500_

### PR DESCRIPTION
**Pull request by Akshat2Jain**
## Description:

This PR updates the documentation for the Pluggables framework to provide a clearer understanding of its capabilities and limitations. The changes aim to improve the clarity and accuracy of the information provided to developers.

## Changes Made:

Added a note highlighting that Pluggables are not compatible with server-side rendering (SSR) and are better suited for visual enhancements rather than handling critical data during the loading process.

This Pr fixes #4735

### CODE REVIEW

**Comment by Akshat2Jain** [comment]
The major change is at line no `60` .

File: docs/source/recipes/pluggables.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 myst:
   html_meta:
-    "description": "The Pluggables framework give you insertion points to push components to other components in an 'out of tree' fashion, like React's Portal component, but with vitamins."
-    "property=og:description": "The Pluggables framework give you insertion points to push components to other components in an 'out of tree' fashion, like React's Portal component, but with vitamins."
-    "property=og:title": "Pluggables framework"
-    "keywords": "Pluggables, framework, React, portal"
+    'description': "The Pluggables framework give you insertion points to push components to other components in an 'out of tree' fashion, like React's Portal component, but with vitamins."
+    'property=og:description': "The Pluggables framework give you insertion points to push components to other components in an 'out of tree' fashion, like React's Portal component, but with vitamins."
+    'property=og:title': 'Pluggables framework'
+    'keywords': 'Pluggables, framework, React, portal'
```
Comment (stevepiercy) [inline docs/source/recipes/pluggables.md:7]: Revert the above changes, and configure your editor accordingly.

File: docs/source/recipes/pluggables.md [hunk 2]
```diff
@@ -57,13 +57,15 @@ Internally, the `<PluggablesProvider>` keeps a record of `Pluggables` and `Plug`
 this is achieved by having the `<Pluggables>` and `<Plug>` components register
 themselves with the Provider via React context.
 
+Note: It is important to note that while Pluggables are a powerful framework for enhancing component integration, they are currently not compatible with server-side rendering (SSR). They excel in providing dynamic visual enhancements for CMSUI and client components/widgets. However, when it comes to handling critical data that needs to load quickly, alternative approaches may be more suitable.
```
Comment (stevepiercy) [inline docs/source/recipes/pluggables.md:60]: Use MyST syntax and one sentence per line for docs.

````suggestion
```{note}
While Pluggables are a powerful framework for enhancing component integration, they are currently not compatible with server-side rendering (SSR).
They excel in providing dynamic visual enhancements for the user interface, such as client components and widgets.
However, when it comes to handling critical data that needs to load quickly, alternative approaches may be more suitable.
```
````

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Revert `html_meta` changes and some grammar fixes. Thank you!

**Comment by stevepiercy** [comment]
@Akshat2Jain this PR needs a change log entry: https://6.docs.plone.org/contributing/index.html#change-log-entry

You can ignore the failed docs build, which should be fixed in https://github.com/plone/volto/pull/4941

**Comment by stevepiercy** [comment]
@Akshat2Jain you need to merge the latest changes from `master` into this branch. There should be a button in the GitHub UI. The failing docs check will pass after you do so.

Screen Shot 2023-07-08 at 1 14 24 AM

File: docs/source/recipes/pluggables.md [hunk 3]
```diff
@@ -57,7 +57,15 @@ Internally, the `<PluggablesProvider>` keeps a record of `Pluggables` and `Plug`
 this is achieved by having the `<Pluggables>` and `<Plug>` components register
 themselves with the Provider via React context.
 
-Note: It is important to note that while Pluggables are a powerful framework for enhancing component integration, they are currently not compatible with server-side rendering (SSR). They excel in providing dynamic visual enhancements for CMSUI and client components/widgets. However, when it comes to handling critical data that needs to load quickly, alternative approaches may be more suitable.
+```{note}
+
```
Comment (stevepiercy) [inline docs/source/recipes/pluggables.md:61]: Remove extra lines. A blank line causes a new paragraph.

```suggestion
```

File: docs/source/recipes/pluggables.md [hunk 4]
```diff
@@ -57,7 +57,15 @@ Internally, the `<PluggablesProvider>` keeps a record of `Pluggables` and `Plug`
...
+While Pluggables are a powerful framework for enhancing component integration, they are currently not compatible with server-side rendering (SSR).
+
```
Comment (stevepiercy) [inline docs/source/recipes/pluggables.md:63]:
```suggestion
```

File: docs/source/recipes/pluggables.md [hunk 5]
```diff
@@ -57,7 +57,15 @@ Internally, the `<PluggablesProvider>` keeps a record of `Pluggables` and `Plug`
...
+They excel in providing dynamic visual enhancements for the user interface, such as client components and widgets.
+
```
Comment (stevepiercy) [inline docs/source/recipes/pluggables.md:65]:
```suggestion
```

File: docs/source/recipes/pluggables.md [hunk 6]
```diff
@@ -57,7 +57,15 @@ Internally, the `<PluggablesProvider>` keeps a record of `Pluggables` and `Plug`
...
+However, when it comes to handling critical data that needs to load quickly, alternative approaches may be more suitable.
+
+	```
```
Comment (stevepiercy) [inline docs/source/recipes/pluggables.md:68]: This closing fence must be at the same indentation as the opening fence.

````suggestion
```
````

**Review by stevepiercy — CHANGES_REQUESTED** [review]
A few quick and easy MyST fixes, and it is good to go!

FYI you can preview the built docs either by doing `make docs-html` or `make docs-livehtml` or looking at it in the [Netlify link in the pull request in GitHub](https://github.com/plone/volto/pull/4948#issuecomment-1621562873).

File: news/4932.bugfix
Comment (stevepiercy) [inline news/4932.bugfix:1]: Do not delete change log entries. They will be removed by the release manager. Please restore this file.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Docs changes look good. However, I just noticed that you should restore the deleted change log entry.

[news/4932.bugfix](https://github.com/plone/volto/pull/4948/files#diff-876681cc6263aed16b184ad56486b48cd44c06efbedc27cd39f4ea3418c9d75b)

File: news/4932.bugfix [hunk 7]
```diff
@@ -1 +1 @@
-Add a marker in the props passed to `RenderBlocks` in the Grid block view @sneridagh
+Add a marker in the props passed to `RenderBlocks` in the Grid block view @sneridagh
```
Comment (stevepiercy) [inline news/4932.bugfix:1]: Remove trailing space.

```suggestion
Add a marker in the props passed to `RenderBlocks` in the Grid block view @sneridagh
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Almost there! This tidies up the change log.

**Review by stevepiercy — APPROVED** [review]
Thank you! If it is all green, then it can be merged.

**Comment by stevepiercy** [comment]
@sneridagh this is docs only, and I approve. I'll let you merge and release.

**Review by sneridagh — APPROVED** [review]

**Pull request merged**